src/main/java/seedu/duke/TeamMemberList.java

```diff
@@ -0,0 +1,38 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+
+public class TeamMemberList {
+    protected static ArrayList<TeamMember> memberList = new ArrayList<>();
+
+    public TeamMemberList() {
+
+    }
+
+    public static void addTeamMember(String name) {
+        TeamMember newMember = new TeamMember(name);
+        memberList.add(newMember);
+    }
+
+    public static void removeTeamMember(String name) {
+        memberList.removeIf(member -> (member.name.contains(name)));
+    }
+
+    public static void removeTeamMember(Integer id) {
+        if (id < memberList.size()) {
+            memberList.remove(id - 1);
+        }
+    }
+
+    public static String listTeamMembers() {
+        String listOfMembers = "";
```

Comment:
Should we account for zero team members? Catch an error?

Reply:
thanks for pointing that out sam ! fixed here 7cd85a2268c4e2eeb93a769ebcb82c17dd9f30f9